### DISCUSSION TITLE
数据源加入Doris

### DISCUSSION THREAD

**Discussion by weakdouqing** [apache/seatunnel#5666] [Q&A]
我想知道将Doris作为数据源是否已经列入了计划？
很喜欢seatunnel这个组件，但当我想要从Doris抽数据迁移到其他数据库中时，发现数据源并没有Doris，因此想问一下是否已经将Doris作为数据源加入了开发计划或者后续不会考虑在数据源中加入Doris

**Comment by liunaijie** [comment]
It's supported now
